modules/calc/src/main/java/com/opengamma/strata/calc/runner/ResultsListener.java

```diff
@@ -32,16 +32,16 @@
   private final List<CalculationResult> results = new ArrayList<>();
 
   /** The columns that define what values are calculated. */
-  private final List<Column> columns;
+  private List<Column> columns;
```

Comment:
The calling code goes to great lengths to ensure everything is synchronised so all changes are visible to all threads. The Javadoc on `CalculationListener` specifies that implementations don't have to be thread safe.